tests/deployment/test_client_server.py

```diff
@@ -255,28 +247,26 @@ def client_server_simulation(x_train, x_test, model, default_configuration):
 
     # Back to the client
 
-    # Decrypt and de-quantize the result
+    # Decrypt, de-quantize and post-processed the result
     y_pred_on_client_quantized = fhemodel_client.deserialize_decrypt(serialized_result)
     y_pred_on_client_dequantized = fhemodel_client.deserialize_decrypt_dequantize(serialized_result)
 
     # Get the y_pred_model_server_clear
 
     # Predict based on the model we are testing
     qtest = model.quantize_input(x_test)
-    y_pred_model_server_ds_quantized = model.fhe_circuit.encrypt_run_decrypt(qtest)
```

Comment:
y_pred_model_server_ds_quantized are neither the outputs of the client nor the outputs of the server. But the outputs of the dev model, right ?

Reply:
output of the dev circuit yes